extensions/micrometer/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,11 @@
+name: "Micrometer - SLF4J for metrics"
+metadata:
+  keywords:
+  - "micrometer"
+  - "metrics"
+  - "metric"
+  - "prometheus"
+  guide: "https://github.com/ebullient/quarkus-micrometer-extension"
```

Comment:
Can't remember if we'd discussed or not, did you want to include a guide with this PR?

Reply:
Yes. I can and should.

Reply:
Latest commit has the beginnings of a guide. We should talk about what else should be in there..